__tests__/app.test.js

```diff
@@ -37,6 +37,98 @@ describe("app", () => {
       });
     });
     describe("/reviews", () => {
+      describe("GET: /api/reviews", () => {
+        test("200: responds with an array of review objects when not passed query", () => {
+          return request(app)
+            .get("/api/reviews")
+            .expect(200)
+            .then(({ body: reviews }) => {
+              expect(reviews).toHaveLength(13);
+
+              reviews.forEach((review) => {
+                expect(review).toEqual(
+                  expect.objectContaining({
+                    owner: expect.any(String),
+                    title: expect.any(String),
+                    review_id: expect.any(Number),
+                    category: expect.any(String),
+                    review_img_url: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    designer: expect.any(String),
+                    comment_count: expect.any(Number),
+                  })
+                );
+              });
+            });
+        });
+        test("200: responds with an array of review objects sorted by date in descending order when not passed query", () => {
+          return request(app)
+            .get("/api/reviews")
+            .expect(200)
+            .then(({ body: reviews }) => {
+              expect(reviews).toHaveLength(13);
+
+              const sortedReviews = reviews.map((review) => {
+                return { ...review };
+              });
+
+              function compareDates(a, b) {
+                if (a.created_at > b.created_at) {
+                  return -1;
+                }
+                if (a.created_at < b.created_at) {
+                  return 1;
+                }
+                return 0;
+              }
+
+              sortedReviews.sort(compareDates);
```

Comment:
Jest-sorted could have saved you a bit of faff here with but it works so I'm happy!